tests/acceptance/cli_test.py

```diff
@@ -1141,6 +1168,47 @@ def test_unpause_no_containers(self):
     def test_logs_invalid_service_name(self):
         self.dispatch(['logs', 'madeupname'], returncode=1)
 
+    def test_logs_follow(self):
+        self.base_dir = 'tests/fixtures/logs-composefile'
+        self.dispatch(['up', '-d'], None)
+
+        logs_proc = start_process(self.base_dir, ['logs', '-f'])
+        condition = NumberOfLinesCondition(logs_proc, 4)
+        wait_on_condition(condition)
+        assert logs_proc.poll() is None
+
+        os.kill(logs_proc.pid, signal.SIGINT)
+        result = condition.get_lines()
+
+        assert 'hello' in result
+        assert 'test' in result
+        service = self.project.get_service('simple')
+        self.assertEqual(len(service.containers()), 1)
```

Comment:
Minor: we've been moving away from these `unittest` assertions, to use bare `assert` statements instead, which work much better with `pytest`. It would be good to just make this `assert len(...) == 1`